public/app/features/dashboard/services/TimeSrv.ts

```diff
@@ -275,7 +275,9 @@ export class TimeSrv {
   // resume auto-refresh based on old dashboard refresh property
   resumeAutoRefresh() {
     this.autoRefreshPaused = false;
-    this.refreshTimeModel();
+    if (this.timeModel?.refresh) {
+      this.refreshTimeModel();
+    }
```

Comment:
this also seems wrong, we should only trigger refresh if the time has changed while in panel edit (and should have nothing to do with auto refresh).

Reply:
Updated the logic so that if the time range is updated while in panel edit, or if the auto refresh interval is hit while in panel edit, refresh will be triggered when panel edit is exited.